content/source/docs/enterprise/install/automating-the-installer.html.md

```diff
@@ -144,7 +147,7 @@ The following settings apply to every installation:
 
 - `hostname` — (Required) The hostname you will use to access your installation.
 - `installation_type` — (Required) One of `poc` or `production`.
-- `enc_password` — Set the [encryption password](./encryption-password.html) for the install.
+- `enc_password` — (Required) The [password](./encryption-password.html) used to encrypt and decrypt the internally-managed Vault unseal key and root token. Not required only when opting out of internally-managed Vault.
```

Comment:
```suggestion
- `enc_password` — (Required) The [password](./encryption-password.html) used to encrypt and decrypt the internally managed Vault unseal key and root token. Not required only when opting out of internally-managed Vault.
```

I'm commenting on this after my comment lower down, so I'll not repeat myself too much - I'm not sure we need this hyphenated.